### PR TITLE
fix(setting-panel): add meta for setting panel

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
  - Introduced advanced metadata settings for events, properties, and styles.

- **Refactor**
  - Reorganized settings to utilize a centralized metadata file for events, properties, and styles.
  - Simplified and standardized the structure of settings exports for better maintainability.

### CODE REVIEW

**Comment by chilingling** [comment]
@coderabbitai review

File: packages/settings/events/meta.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default {
+  id: 'engine.setting.SettingEvents',
```
Comment (gene9831) [inline packages/settings/events/meta.js]: 根据下面其他id格式，应该改成 `engine.setting.events`

Comment (chilingling) [reply packages/settings/events/meta.js]: Done

**Comment by rhlin** [comment]
需要添加@pentiny/tiny-engine-entry的包 所有包都需要

**Review by hexqi — APPROVED** [review]

**Pull request merged**